src/zenml/integrations/azure/orchestrators/azureml_orchestrator.py

```diff
@@ -506,14 +506,16 @@ def fetch_status(self, run: "PipelineRunResponse") -> ExecutionStatus:
             return ExecutionStatus.INITIALIZING
         elif status in ["Running", "Finalizing"]:
             return ExecutionStatus.RUNNING
+        elif status == "CancelRequested":
+            return ExecutionStatus.STOPPING
+        elif status == "Canceled":
+            return ExecutionStatus.STOPPED
```

Comment:
Same here: there is no implementation for `_stop` in this orchestrator.

Reply:
Same as above, so I am resolving this one for now.